SPECS/python-requests/CVE-2024-35195.patch

```diff
@@ -0,0 +1,125 @@
+diff --git a/requests/adapters.py b/requests/adapters.py
```

Comment:
I would add a comment on where the patch was takens from, my understanding is that it should be like:
```suggestion
Patch taken from https://github.com/psf/requests/commit/a58d7f2ffb4d00b46dca2d70a3932a0b37e22fac
diff --git a/requests/adapters.py b/requests/adapters.py
```

Reply:
Fixed, thanks.